ddp-workspace/projects/toolkit/src/lib/components/dialogs/sessionWillExpire.component.ts

```diff
@@ -26,7 +31,7 @@ import { interval, Subscription } from 'rxjs';
                 [disabled]="isRenewing"
                 [innerHTML]="'Toolkit.Dialogs.SessionWillExpire.SignOut' | translate">
         </button>
-        <button class="ButtonFilled Button--rect button button_small button_primary"
+        <button *ngIf="!renewalFailed" class="ButtonFilled Button--rect button button_small button_primary"
                 (click)="renewSession()"
                 [disabled]="isRenewing"
                 [innerHTML]="'Toolkit.Dialogs.SessionWillExpire.Continue' | translate">
```

Comment:
Updated dialog to display message about renewal failing. The logout button is the only option under this condition.
It is kind of inflexible and a little inconvenient if it happens, but I expect this is a rare situation. And no data should be lost. And no bad data is written. But welcome input.

What dialog looks like in Osteo (pretty similar in Brain):
<img width="1156" alt="Screen Shot 2020-04-25 at 10 56 47 AM" src="https://user-images.githubusercontent.com/29984380/80283153-a0b2ec00-86e3-11ea-9715-58e5f432aa57.png">